testdata/cilium/expected/expectedCilium.py

```diff
@@ -290,8 +290,8 @@
                             "sh",
                             "-ec",
                             """cp /usr/bin/cilium-sysctlfix /hostbin/cilium-sysctlfix;
-              nsenter --mount=/hostproc/1/ns/mnt "${BIN_PATH}/cilium-sysctlfix";
-              rm /hostbin/cilium-sysctlfix
+nsenter --mount=/hostproc/1/ns/mnt "${BIN_PATH}/cilium-sysctlfix";
+rm /hostbin/cilium-sysctlfix
```

Comment:
@rquitales we get an unintended benefit of retrieving the true value of the multi-line string, without the trailing whitespace on each line.

Reply:
Nice!